testing/pdal/APKBUILD

```diff
@@ -11,8 +11,7 @@ depends="nitro python py-numpy"
 makedepends="cmake eigen-dev hexer-dev nitro-dev gdal-dev geos-dev laz-perf-dev libgeotiff-dev libxml2-dev python-dev py-numpy-dev jsoncpp-dev hdf5-dev proj4-dev cpd-dev fgt-dev sqlite-dev postgresql-dev curl-dev laszip-dev libspatialite-dev linux-headers libexecinfo-dev"
 install=""
 subpackages="$pkgname-dev"
-source="$pkgname-$pkgver.tar.gz::http://download.osgeo.org/$pkgname/PDAL-$pkgver-src.tar.gz
-	fix-shactx-bitlen.patch"
```

Comment:
You missed to remove the patch `fix-shactx-bitlen.patch` in the same commit

Reply:
Thanks for catching @andypost. Fixed!